enhancements/openshift-route-admission-policy.md

```diff
@@ -77,16 +75,48 @@ Addition of the new field requires no migration, as the default matches the curr
 
 ```go
 // RouteAdmissionPolicy is an admission policy for allowing new route claims.
-type RouteAdmissionPolicy string
+type RouteAdmissionPolicy struct {
+  // namespaceOwnership describes how host name claims across namespaces should
+  // be handled. The default is Strict.
+  //
+  // Value must be one of:
+  //
+  // - Strict: do not allow routes to claim the same host name across namespaces.
+  // - InterNamespaceAllowed: allow routes to claim different paths of the same host name across namespaces.
+  //
+  // The default is Strict.
```

Comment:
Is there a test that ensures pruning is enabled in 4.3 so that it is impossible for you to get this stanza with an empty `namespaceOwnershipCheck` during an upgrade where you have skewed CRDs and ingress controller?  The same issue comes up on downgrades.

Alternatively (or perhaps in addition), if this does appear and is empty, the ingress controller must treat it as strict.